platform/platform.json

```diff
@@ -3,9 +3,6 @@
   "mobileAppVersion": "1.7.2-rc.6",
   "releaseNotes": "* Various iPhone X styling improvements\n* Status bar color picker for style customization\n* Bugfixes and improvements",
   "settings": {
```

Comment:
Wouldn't it make more sense to remove the entire `settings` property? Or is that not possible with the API request?

Reply:
It would, I didn't want to remove it since server validation expects them to be non-empty as well. I could be persuaded to remove that validation too, though. But it might be nice for people to have blank settings there so they know they can put things in there. I'm not sure but it's possible that settings get copied over to platform installation, and that mechanism could be used by platform developers to save various things.